core/commands/root.go

```diff
@@ -69,6 +69,9 @@ TOOL COMMANDS
     commands      List all available commands
 
 Use 'ipfs <command> --help' to learn more about each command.
+
+If you want to operate on a different repository than the default at ~/.ipfs,
+then prefix any ipfs command with IPFS_PATH=/path.
```

Comment:
I would phrase this a little different, to give the reader a more clear idea of whats going on:

"You may set the `IPFS_PATH` environment variable to make ipfs use a different location for its on disk repo."

Or something, cc @jbenet @RichardLitt